lab-michelle/src/component/dashboard-container/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {
+  categoryDestroy, categoryCreate, categoryUpdate,
+} from '../../action/category-actions';
+import CategoryForm from '../category-form';
+import CategoryItem from '../category-item';
+
+
+class DashboardContainer extends React.Component {
+  componentDidMount() {
+    console.log('__DASHBOARD__', this);
+  }
+
+  render() {
+    return(
+      <main className="main-content">
+        <h2>Dashboard</h2>
+
+        <CategoryForm
```

Comment:
this component should support a category prop that will initialize the state of the form